src/Microsoft.ML.StaticPipe/NormalizerStaticExtensions.cs

```diff
@@ -82,98 +82,98 @@ public static class NormalizerStaticExtensions
         /// to outliers as compared to <see cref="Normalize(Vector{float}, bool, long, OnFitAffine{ImmutableArray{float}})"/>.
         /// </summary>
         /// <param name="input">The input column.</param>
-        /// <param name="fixZero">If set to <c>true</c> then the offset will always be considered zero.</param>
+        /// <param name="ensureZeroUntouched">If set to <c>true</c> then the offset will always be considered zero.</param>
         /// <param name="useLog">If set to true then we transform over the logarithm of the values, rather
-        /// than just the raw values. If this is set to <c>true</c> then <paramref name="fixZero"/> is ignored.</param>
-        /// <param name="maxTrainingExamples">When gathering statistics only look at most this many examples.</param>
+        /// than just the raw values. If this is set to <c>true</c> then <paramref name="ensureZeroUntouched"/> is ignored.</param>
+        /// <param name="maximumExampleCount">When gathering statistics only look at most this many examples.</param>
         /// <param name="onFit">A delegate called whenever the estimator is fit, with the learned slopes
-        /// and, if <paramref name="fixZero"/> is <c>false</c>, the offsets as well.</param>
+        /// and, if <paramref name="ensureZeroUntouched"/> is <c>false</c>, the offsets as well.</param>
         /// <remarks>Note that the statistics gathering and normalization is done independently per slot of the
         /// vector values.</remarks>
         /// <returns>The normalized column.</returns>
-        public static NormVector<float> NormalizeByMeanVar(
-            this Vector<float> input, bool fixZero = FZ, bool useLog = false, long maxTrainingExamples = MaxTrain,
+        public static NormVector<float> NormalizeByMeanVariance(
+            this Vector<float> input, bool ensureZeroUntouched = FZ, bool useLog = false, long maximumExampleCount = MaxTrain,
             OnFitAffine<ImmutableArray<float>> onFit = null)
         {
-            return NormalizeByMVCdfCore(input, fixZero, useLog, false, maxTrainingExamples, AffineMapper(onFit));
+            return NormalizeByMVCdfCore(input, ensureZeroUntouched, useLog, false, maximumExampleCount, AffineMapper(onFit));
         }
 
         /// <summary>
         /// Learns an affine function based on the observed mean and standard deviation. This is less susceptible
         /// to outliers as compared to <see cref="Normalize(Vector{double}, bool, long, OnFitAffine{ImmutableArray{double}})"/>.
         /// </summary>
```

Comment:
This should be expanded upon at some time. Maybe in the docs go-around.

Reply:
Totally agree but not this huge PR.

---
In reply to: [264868709](https://github.com/dotnet/machinelearning/pull/2888#discussion_r264868709) [](ancestors = 264868709)